assets/php/virion.php

```diff
@@ -139,78 +135,41 @@ function change_dna(string $chromosome, string $antigen, string $antibody, $mode
         case VIRION_INFECTION_MODE_SYNTAX:
             $tokens = token_get_all($chromosome);
             $tokens[] = ""; // should not be valid though
-            if(PHP_VERSION_ID >= 80000){
-                //PHP8 Specific, https://wiki.php.net/rfc/namespaced_names_as_token
-                foreach($tokens as $offset => $token) {
-                    if(!is_array($token) or $token[0] !== T_WHITESPACE) {
-                        /** @noinspection IssetArgumentExistenceInspection */
-                        list($id, $str, $line) = is_array($token) ? $token : [-1, $token, $line ?? 1];
-                        //namespace test; is a T_STRING whereas namespace test\test; is not.
-                        if(isset($init, $prefixToken) and $id === T_STRING){
-                            if(strpos($str, $antigen) === 0) { // case-sensitive!
-                                $tokens[$offset][1] = $antibody . substr($str, strlen($antigen));
-                                ++$count;
-                            } elseif(stripos($str, $antigen) === 0) {
-                                echo "\x1b[38;5;227m\n[WARNING] Not replacing FQN $str case-insensitively.\n\x1b[m";
-                            }
-                            unset($init, $prefixToken);
-                        } else {
-                            if($id === T_NAMESPACE) {
-                                $init = $offset;
-                                $prefixToken = $id;
-                            } elseif($id === T_NAME_QUALIFIED) {
+            foreach($tokens as $offset => $token) {
+                if(!is_array($token) or $token[0] !== T_WHITESPACE) {
+                    /** @noinspection IssetArgumentExistenceInspection */
+                    list($id, $str, $line) = is_array($token) ? $token : [-1, $token, $line ?? 1];
+                    //namespace test; is a T_STRING whereas namespace test\test; is not.
+                    if(isset($init, $prefixToken) and $id === T_STRING){
+                        if($str === $antigen) { // case-sensitive!
```

Comment:
Why? Does this work with subnamespaces under the antigen?

Reply:
please stop using these ridiculous names, nobody has any idea what you're talking about